client/src/rpc.rs

```diff
@@ -278,6 +280,11 @@ impl EthRpcServer for RpcInner {
         Ok(node.get_coinbase().unwrap())
     }
 
+    async fn syncing(&self) -> Result<SyncingStatus, Error> {
+        let node = self.node.read().await;
+        Ok(node.syncing().unwrap())
```

Comment:
Can you handle the error here instead of unwrapping?  See how many of the other methods use `convert_err(some_result)?` for how we handle it.

In general we want to avoid ever using the `unwrap` method unless it is explicitly checked, and even then it is more idiomatic to use something like the `if let` syntax, result mapping, `ok_or`, or pattern matching.

Reply:
Got it. Thank you